src/structures/MessageEmbed.js

```diff
@@ -108,6 +110,39 @@ class MessageEmbed {
     while (col.length < 6) col = `0${col}`;
     return `#${col}`;
   }
+
+  /**
+   * The RichEmbed representation of this MessageEmbed.
+   * @type {RichEmbed}
+   * @readonly
+   */
+  get richEmbed() {
+    const result = {};
+    for (const key of Object.keys(this)) {
```

Comment:
This will include the `message` property and throw later on this: `TypeError: Cannot convert undefined or null to object`.

`if (key === 'message') continue;` could prevent that.
It would also be possible to delete the property, or set it to null / undefined.
I am not sure what the best way would be, probably continue.